content/docs/pulumi-cloud/neo/_index.md

```diff
@@ -4,6 +4,9 @@ meta_desc: Pulumi Neo is an agentic AI that enables conversational infrastructur
 title: Pulumi Neo
 h1: Pulumi Neo
 meta_image: /images/docs/meta-images/docs-meta.png
+aliases:
+- /docs/iac/neo/
+- /docs/neo/
 menu:
```

Comment:
Really feels like something _should_ need to edited here in order to make it a child of the Pulumi Cloud menu, but the preview is clearly correct.